cypress/integration/navigationFlows/homePageLeftSidebar.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Home Page Left Sidebar', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should open the "More" links', () => {
+    // Go to home page
+    cy.visit('/');
+
+    // Click on the More... button in the nav
+    cy.get('#page-content-inner').within(() => {
+      cy.findByText('More...').click();
+      cy.findByText('More...').should('have.class', 'hidden');
+      cy.findByText('Nav link 5').should('not.have.class', 'hidden');
```

Comment:
```suggestion
      cy.findByText('More...').should('not.be.visible');
      cy.findByText('Nav link 5').should('be.visible');
```

(blocking) - I think we should switch up the assertion here - Cypress can detect whether or not the element is currently visible, so rather than checking for the classname that should _produce_ that effect, we can just check the end effect without worrying about the implementation 🙂